### PR TITLE
fix: set USE_DOCSTRING as default for ai_callable

### PR DESCRIPTION
**Pull request by longcw**
`@llm.ai_callable` need a explicit `description=llm.USE_DOCSTRING` to use the docstring as the description.

### CODE REVIEW

**Review by davidzhao — COMMENTED** [review]
this is rather annoying tho as a requirement.. wdyt about just making that the default behavior?

if no explicit description is provided, it'll use docstring. if docstring isn't set, it raises

**Comment by longcw** [comment]
I think that makes sense, most people will explicitly set a description or write that in the docstring.

At least the current example is very misleading, I've always assumed that I just need to set the description at docstring, but it actually also needs `description=llm.USE DOCSTRING` to take effect.

**Comment by davidzhao** [comment]
> I think that makes sense, most people will explicitly set a description or write that in the docstring.
> 
> At least the current example is very misleading, I've always assumed that I just need to set the description at docstring, but it actually also needs `description=llm.USE_DOCSTRING` to take effect.

honestly I did too.. 🤦  I think it should just "do the obvious thing" here..

**Comment by davidzhao** [comment]
@longcw do you wanna take a stab at making it the default behavior?

**Comment by longcw** [comment]
@davidzhao sure, updated.

File: livekit-agents/livekit/agents/llm/function_context.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ class CalledFunction:
 def ai_callable(
     *,
     name: str | None = None,
-    description: str | _UseDocMarker | None = None,
+    description: str | _UseDocMarker | None = USE_DOCSTRING,
```
Comment (davidzhao) [inline livekit-agents/livekit/agents/llm/function_context.py:108]: we don't need the `None` value anymore? it seems that we need either a description, or USE_DOCSTRING

**Review by davidzhao — APPROVED** [review]

**Pull request merged**